build/install-operator-sdk.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+if [ -z "${OPERATOR_SDK_VERSION}" ]; then
+    echo "OPERATOR_SDK_VERSION should be defined as an env varialbe (usually in makefile)"
+    exit 1
+fi
+
+if [ -z "${OPERATOR_SDK}" ]; then
+    echo "OPERATOR_SDK should be defined as an env varialbe (usually in makefile)"
+    exit 1
+fi
+
+mkdir -p $(dirname "${OPERATOR_SDK}")
+if [ -x ${OPERATOR_SDK} ]
+then
+    ${OPERATOR_SDK} version | grep -q "\"${OPERATOR_SDK_VERSION}\""
+    if [ $? -eq 0 ]
+    then
+        exit 0
+    fi
+fi
+
+PLATFORM=$(uname)
+if [ "${PLATFORM}" == "Darwin" ] 
+then 
+    SDK_RELEASE="https://github.com/operator-framework/operator-sdk/releases/download/${OPERATOR_SDK_VERSION}/operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-apple-darwin"
+else 
+    # Assuming that if not darwin then running on linux
```

Comment:
Well, uname on Linux returns 'Linux'.

Reply:
Which also gets into the else, do you see a problem here?
We don't test compiling over WIN, but that's the only issue I see with a generic else ....

Reply:
I don't see a problem, was just stating that instead of assuming we can use 'Linux' just as we test for 'Darwin', that's all.